index.js

```diff
@@ -0,0 +1,106 @@
+const assert = require('assert');
+const debug = require('debug')('rubric');
```

Comment:
Not sure if you had some plans for browser bundling to mitigate this issue - just noting that this would add a few kB to the bundle.